ReadMe.md

```diff
@@ -28,8 +28,8 @@ Some handy links:
 
 Support for multiplatform programming is one of Kotlin’s key benefits. It reduces time spent writing and maintaining the same code for [different platforms](https://kotlinlang.org/docs/reference/mpp-supported-platforms.html) while retaining the flexibility and benefits of native programming.
 
- * [Kotlin Multiplatform Mobile](https://kotlinlang.org/lp/mobile/) for sharing code between Android and iOS
- * [Getting Started with Kotlin Multiplatform Mobile Guide](https://kotlinlang.org/docs/mobile/create-first-app.html)
+ * [Kotlin Multiplatform](https://kotlinlang.org/lp/mobile/) for sharing code between Android and iOS
```

Comment:
The link should be replaced as well: https://www.jetbrains.com/kotlin-multiplatform/
Also the description should be: `for sharing code between Android, iOS, web, desktop, and server-side`